Instabile Frontendtests in wls-gui

Workflow wo es zu dem Fehler kam:

https://github.com/it-at-m/Wahllokalsystem/actions/runs/16443808828/job/46470579165

<details>

<summary>Log</summary>

````log
stderr | tests/composables/useCurrentTime.spec.ts > useCurrentTime > should_updateCurrentTime_when_timeIncrementsForOneSecond
[Vue warn]: onBeforeUnmount is called when there is no active component instance to be associated with. Lifecycle injection APIs can only be used during execution of setup(). If you are using async setup(), make sure to register lifecycle hooks before the first await statement.

 ✓ tests/composables/useCurrentTime.spec.ts (1 test) 6ms
⎯⎯⎯⎯⎯⎯ Unhandled Errors ⎯⎯⎯⎯⎯⎯

Vitest caught 1 unhandled error during the test run.
This might cause false positive tests. Resolve unhandled errors to make sure your tests are not affected.

⎯⎯⎯⎯ Unhandled Rejection ⎯⎯⎯⎯⎯
ReferenceError: document is not defined
 ❯ S node_modules/vue3-toastify/dist/index.mjs:772:5
    770|   }
    771|   function S() {

    772|     document.hasFocus() || h(), window.addEventListener("focus", I), w…
       |     ^
    773|   }
    774|   function _() {
 ❯ m node_modules/vue3-toastify/dist/index.mjs:755:27
 ❯ node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:2818:40
 ❯ callWithErrorHandling node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:200:19
 ❯ callWithAsyncErrorHandling node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:207:17
 ❯ hook.__weh.hook.__weh node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:2798:19
 ❯ flushPostFlushCbs node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:382:28
 ❯ flushJobs node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:424:5

This error originated in "tests/components/wahlvorbereitung/BaseWahleroeffnungCard.spec.ts" test file. It doesn't mean the error was thrown inside the file itself, but while it was running.
This error was caught after test environment was torn down. Make sure to cancel any running tasks before test finishes:
- cancel timeouts using clearTimeout and clearInterval
- wait for promises to resolve using the await keyword
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

 Test Files  74 passed (74)
      Tests  608 passed (608)
     Errors  1 error
   Start at  12:08:28
   Duration  38.26s (transform 3.83s, setup 0ms, collect 38.58s, tests 13.02s, environment 39.90s, prepare 10.36s)


Error: ReferenceError: document is not defined
 ❯ S node_modules/vue3-toastify/dist/index.mjs:772:5
 ❯ m node_modules/vue3-toastify/dist/index.mjs:755:27
 ❯ node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:2818:40
 ❯ callWithErrorHandling node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:200:19
 ❯ callWithAsyncErrorHandling node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:207:17
 ❯ hook.__weh.hook.__weh node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:2798:19
 ❯ flushPostFlushCbs node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:382:28
 ❯ flushJobs node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:424:5

This error originated in "tests/components/wahlvorbereitung/BaseWahleroeffnungCard.spec.ts" test file. It doesn't mean the error was thrown inside the file itself, but while it was running.
This error was caught after test environment was torn down. Make sure to cancel any running tasks before test finishes:
- cancel timeouts using clearTimeout and clearInterval
- wait for promises to resolve using the await keyword

Error: Process completed with exit code 1.
```

</details>

Ein retry des Jobs führte zum Erfolg, alle Tests liefen erfolgreich durch.

--- a/wls-gui-wahllokalsystem/tests/components/wahlvorbereitung/BaseWahleroeffnungCard.spec.ts
+++ b/wls-gui-wahllokalsystem/tests/components/wahlvorbereitung/BaseWahleroeffnungCard.spec.ts
@@ -25,6 +25,13 @@ declare module "@vue/runtime-core" {
 
 const mockDefinitions = vi.hoisted(() => ({
   postEroeffnungsuhrzeit: vi.fn(),
+  saveEreignisse: vi.fn(),
+}));
+
+vi.mock("@/composables/vorfaelleundvorkommnisse/ereignisService.ts", () => ({
+  useEreignisService: () => ({
+    saveEreignisse: mockDefinitions.saveEreignisse,
+  }),
 }));
 
 vi.mock("@/composables/wahlvorbereitung/wahlvorbereitungService", () => ({
@@ -265,6 +272,7 @@ describe("BaseWahleroeffnungCard.vue", () => {
           ereignisStore.wahlbezirkEreignisse.ereigniseintraege[0].beschreibung
         ).toBe(begruendung);
       }
+      expect(mockDefinitions.saveEreignisse).toHaveBeenCalled();
     });
   });
 });

